Sources/Security/HTTPRequest+Signing.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  HTTPRequest+Signing.swift
+//
+//  Created by Nacho Soto on 11/16/23.
+
+import CryptoKit
+import Foundation
+
+extension HTTPRequest {
+
+    static func headerParametersForSignatureHeader(headers: Headers) -> String? {
```

Comment:
Hmm this name sounds weird to me... Maybe `headerParametersHeaderForSigning`? (To mimick `postParametersHeaderForSigning`

Reply:
I was trying to avoid the double "header" word in `headerParametersHeaderForSigning`, which I think reads kinda confusingly :/

Reply:
Hmm yeah... Not sure... maybe it's ok as it is.